### PR TITLE
feat(server): dynamically switch node host when current node is down

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/app/models/ckb_sync/api.rb [hunk 1]
```diff
@@ -16,6 +18,28 @@ def initialize
...
+    end
+
+    private
+
+    def reassign_api
+      @api = CKB::API.new(host: available_host)
+    end
+
+    def available_host
+      hosts = %w(http://47.111.175.189:8121 http://47.52.93.135:8121 http://39.97.116.150:8121 http://47.245.31.13:8121 http://47.103.75.2:8121)
```
Comment (classicalliu) [inline server/app/models/ckb_sync/api.rb]: hardcode ?

Comment (shaojunda) [reply server/app/models/ckb_sync/api.rb]: Thank you, fixed.

File: server/config/ckb_node_hosts.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+hosts:
+  - "http://47.111.175.189:8121"
```
Comment (duanyytop) [inline server/config/ckb_node_hosts.yml]: We should not show these node urls to public.

Comment (classicalliu) [reply server/config/ckb_node_hosts.yml]: Remove this file from git

two problems:
1. those nodes are not public, you expose them
2. not easy to run another explorer for another chain

**Pull request closed without merging**